### DISCUSSION TITLE
How to scan and show CVE's and tag as "Red: Vulnerable"?

### DISCUSSION THREAD

**Discussion by nocexpedit** [cea-sec/ivre#1481] [General]
Hello, 

If I import nmap vulners results, then it is shown in IVRE but the script "vunlers" is not setup to be auto tagged in red "Vulnerable".

![image](https://user-images.githubusercontent.com/121177782/215458187-67fb85a7-7c75-4d3e-b49d-ff21e30dbd14.png)

How can I tag results that include "vulners" results with the "Vulnerable" label?
![image](https://user-images.githubusercontent.com/121177782/215459327-f97c6e65-a263-41cb-b528-b32577ff9960.png)

Thank you for this great project!

Best regards,
Simo

**Comment by p-l-** [comment]
Hi there,

Thanks for your interest in IVRE. This is not (yet) implemented. You should open an issue to ask for that feature. Aslo, if you could provide Nmap XML results including `vulners` results, that would probably help.

Also, which minimum CVSS score should we keep? Should we require a known exploit to exists?

Pierre

**Comment by nocexpedit** [comment]
Bonjour Pierre,

Thank you for the answer. I will open an issue in which I will ask for that feature.
Minimum CVSS 6.0 if I would use it. 
All exploits.

Best regards,
Simo
